implants/eldritch/src/pivot.rs

```diff
@@ -10,24 +10,35 @@ mod bind_proxy_impl;
 use derive_more::Display;
 
 use starlark::environment::{Methods, MethodsBuilder, MethodsStatic};
-use starlark::values::{StarlarkValue, Value, UnpackValue, ValueLike};
 use starlark::values::none::NoneType;
+use starlark::values::{StarlarkValue, Value, UnpackValue, ValueLike, ProvidesStaticType};
 use starlark::{starlark_type, starlark_simple_value, starlark_module};
 
-#[derive(Copy, Clone, Debug, PartialEq, Display)]
+use serde::{Serialize,Serializer};
+
+#[derive(Copy, Clone, Debug, PartialEq, Display, ProvidesStaticType)]
```

Comment:
`ProvidesStaticType`
Is no a required implementation for modules.
Using the derive is an easy way to add it.